exporter/collector/metrics.go

```diff
@@ -107,6 +107,12 @@ const (
 	sendBatchSize = 200
 )
 
+const (
+	// These are special keys that can be used to export custom supported types for google cloud metrics.
+	gcpCustomType  string = "gcp.cloud_monitoring.custom.type"
+	gcpCustomValue string = "gcp.cloud_monitoring.custom.value"
```

Comment:
I think we could shorten these keys to `gcp.metric_type` or something. We already one magic attribute like this: https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/tree/main/exporter/collector#multi-project-exporting

Reply:
How about `gcp.mapped.metric_type` ? 

I was actually trying to follow naming conventions in [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/cloud_provider/gcp/cloud_run.md) in anticipation that these keys might end up there later.

Reply:
With the new design decisions this conversation is moot now.